shared/oae/css/oae.components.css

```diff
@@ -194,6 +194,85 @@ i.fa.large:before {
 }
 
 
+/****************
+ ** SWITCH TAB **
+ ****************/
+
+.oae-switchtab-container {
+    text-align: center;
+}
+
+.oae-switchtab-container .oae-switchtab-control {
+    background-color: #333;
```

Comment:
All/most of the colors in this component should probably come from the skin file. We could either set up a new section for the switchtab component, or we could leverage some of the existing skin variables:

For example:

- We could use the `oae-tile-metadata` background for the black background. This would then make it align with the tiles
- We could use `.oae-tile > .oae-tile-metadata small` for the text color on the black background. This would then make it align with the tiles
- We could use `.oae-large-options-container label.checked .oae-large-options` for the icon color of the selected tab. This would then make it align with the institutional colors in the visibility options.

I'm not entirely sure which approach I actually prefer. The former approach would be clear and easy to control, but would add another component to the skin and would require us to update all of our existing tenancies. The latter approach would make the institutional colors trickle through to this component without additional work, but could be hard to control.